### PR TITLE
OADP-664: OADP-on-ODF limitations (backupStorageLocation)

### PR DESCRIPTION
**Pull request by RichardHoch**
OADP 1.1.1; OCP 4.9+

Has passed Dev and QE review.

Resolves https://issues.redhat.com/browse/OADP-664 

Previews:
1. In the general installation section: https://51106--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.html

2. As part of Installing OADP on OCF: https://51106--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.html#oadp-configuring-noobaa-for-dr_installing-oadp-ocs

### CODE REVIEW

**Review by eemcmullan — APPROVED** [review]

**Review by PrasadJoshi12 — APPROVED** [review]
lgtm

**Review by nalhadef — COMMENTED** [review]
Just a couple of comments, @RichardHoch.

File: modules/oadp-configuring-noobaa-for-dr.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+:_content-type: PROCEDURE
+[id="oadp-configuring-noobaa-for-dr_{context}"]
+= Configuring NooBaa for disaster recovery on {rh-storage}
+
+If you use cluster storage for your NooBaa bucket `backupStorageLocation` on {rh-storage}, it is recommended to configure NooBaa as an external object store.
+
+[WARNING]
+====
+Failure to configure NooBaa as an external object store might lead to backups not being available in case of a disaster.
```
[the remark this answers is not in the document]
Comment (RichardHoch) [reply modules/oadp-configuring-noobaa-for-dr.adoc]: So much for the pathetic fallacy...fixed.

**Comment by nalhadef** [comment]
/lgtm

Looks good @RichardHoch. Thanks for making the changes. Feel free to request a merge.

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**